javaagent-api/src/main/java/io/opentelemetry/javaagent/instrumentation/api/concurrent/ExecutorInstrumentationUtils.java

```diff
@@ -45,6 +45,13 @@ protected Boolean computeValue(Class<?> taskClass) {
             return false;
           }
 
+          // OrderedExecutor$1 is a worker class that processes tasks submitted to OrderedExecutor
```

Comment:
Use `enclosingClass == OrderedExecutorFactory` similar to the below instead of this one since it's more readable than a lambda